core/src/main/java/org/infinispan/interceptors/impl/DistCacheWriterInterceptor.java

```diff
@@ -87,6 +88,25 @@ public BasicInvocationStage visitPutKeyValueCommand(InvocationContext ctx, PutKe
    }
 
    @Override
+   public Object visitBackupWriteCommand(InvocationContext ctx, BackupWriteCommand command) throws Throwable {
+      return invokeNext(ctx, command).thenApply((rCtx, rCommand, rv) -> {
+         BackupWriteCommand backupWriteCommand = (BackupWriteCommand) rCommand;
+         if (!isStoreEnabled(backupWriteCommand))
+            return rv;
+
+         //no need to check for proper write. we are remote, in a backup owner.
```

Comment:
that doesn't seem a problem with the Triangle but with our code in general. I'll open a JIRA and fix it later.